src/DssSpell.sol

```diff
@@ -53,63 +52,137 @@ contract DssSpellAction is DssAction {
     // uint256 internal constant X_PCT_RATE      = ;
 
     uint256 internal constant MILLION = 10 ** 6;
+    uint256 internal constant BILLION = 10 ** 9;
+    
+    uint256 internal constant WAD     = 10 ** 18;
+
+    uint256 internal constant PSM_ZERO_BASIS_POINTS = 0;
+    uint256 internal constant PSM_ONE_BASIS_POINT   = 1 * WAD / 10000;
+
+    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+    address internal immutable MCD_PSM_USDC_A           = DssExecLib.getChangelogAddress("MCD_PSM_USDC_A");
+    address internal immutable MCD_PSM_GUSD_A           = DssExecLib.getChangelogAddress("MCD_PSM_GUSD_A");
+    address internal immutable MCD_PSM_PAX_A            = DssExecLib.getChangelogAddress("MCD_PSM_PAX_A");
+
+    // MAINNET SPELL ONLY
+
+    /* GemLike  internal immutable MKR                     = GemLike(DssExecLib.mkr());
+
+    address constant internal LBSBLOCKCHAIN_WALLET      = 0xB83b3e9C8E3393889Afb272D354A7a3Bd1Fbcf5C;
+    address constant internal CONSENSYS_WALLET          = 0xE78658A8acfE982Fde841abb008e57e6545e38b3;
+    address constant internal SES_WALLET                = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
+    address constant internal CES_WALLET                = 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da;
+    address constant internal PHOENIX_LABS_WALLET       = 0xD9847E6b1314f0327F320E43B51ca0AaAD6FF509; */
+    
+    // Monetalis Update - Excess Funds Declaration ONLY ON MAINNET
+    // Poll:  https://vote.makerdao.com/polling/QmfZ2nxw#poll-details
+    // Forum: https://forum.makerdao.com/t/request-to-poll-return-excess-mip65-funds-to-surplus-buffer/20115
+    /* string constant public MIP65 = "Qmf7oGxgVoGKMGkzPi2T6nBSTLgrU5C7jmNqaefjJ52Zup"; */
```

Comment:
I would select a comment style.  I think we generally use `// ` per line rather than the `/* ... */` formatting

Reply:
Implemented this